.github/workflows/build_and_test_with_resty_events.yml

```diff
@@ -1,6 +1,13 @@
-name: Build and test
-
-on: [push, pull_request]
+name: Build and test - with resty_events
```

Comment:
It was confusing on GH Actions site as there were simply two workflows named `Build and test`. I've added this suffix to differentiate between them.